pkg/collector/remoteconfig/remote_config.go

```diff
@@ -0,0 +1,114 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+// Package remoteconfig schedule and unschedule integrations through remote-config
+// using the AGENT_INTEGRATIONS product
+package remoteconfig
```

Comment:
Nit: We already have a lot of packages wanting to name themselves "remoteconfig" (you can see this because you have to immediately do an import rename in cmd where you want to use it) although I admit to not having a better name off the top of my head. 😁